services/index.js

```diff
@@ -1,11 +1,11 @@
 'use strict'
 
 const base = require('../core/base-service')
-const createServiceTester = require('../core/service-test-runner/create-service-tester')
-const ServiceTester = require('../core/service-test-runner/service-tester')
+// const createServiceTester = require('../core/service-test-runner/create-service-tester')
```

Comment:
I'll clean this up once I've verified the service tests are passing with the new import structure